### ISSUE TITLE
ra25 can't find file

### ISSUE THREAD

**Issue by StephenJRead** [emer/leabra#9]
I recently reinstalled everything starting with Go 1.13.5 and then rebuilt ra25. Now when I go to run ra25 I get the following message, despite the fact that the “missing" file is in the same directory as the binary. Ra25 opens, but when I try to initialize it, it crashes.  Is it a problem with the path?  I get the same problem when I try to run the binary from the ra25 examples folder.  It used to work before I reinstalled a version of go that uses go.mod.  

(base) StephenReadsMBP:~ read$ /Users/read/go/bin/ra25 ; exit;
2020/01/12 16:25:48 open random_5x5_25.tsv: no such file or directory
2020/01/12 16:25:48 open random_5x5_25.tsv: no such file or directory
OpenGL version 4.1 INTEL-12.10.14

**Comment by rcoreilly** [comment]
```
(base) StephenReadsMBP:~ read$ /Users/read/go/bin/ra25 ; exit;
```

that looks like you're running the installed executable from the go bin directory -- try running the ./ra25 executable from "go build" in the examples/ra25 dir.

**Comment by StephenJRead** [comment]
I will try again, but my recollection is that when I did go build in the ra25 directory it moved it to the go bin directory.  And it doesn’t run in either directory, even if the relevant file is in the directory.  But I will throw RA25 away and try again.


On Jan 14, 2020, at 11:51 AM, rcoreilly <notifications@github.com<mailto:notifications@github.com>> wrote:


(base) StephenReadsMBP:~ read$ /Users/read/go/bin/ra25 ; exit;


that looks like you're running the installed executable from the go bin directory -- try running the ./ra25 executable from "go build" in the examples/ra25 dir.

**Comment by StephenJRead** [comment]
I discovered the problem.  First, I realized that if I double click on the executable it can’t find the needed Data file.  However, if I use the ./ra25 command after I build it, then it works properly. Second, I was doing all this from within the emergent installation in Go and I discovered that at some point all the folders within the emer had been set to read only.  When I changed all the items in the ra25 directory to Read and Write then everything built properly.  Although I still could not start by double clicking the executable.

On Jan 14, 2020, at 11:54 AM, Stephen Read <read@usc.edu<mailto:read@usc.edu>> wrote:

I will try again, but my recollection is that when I did go build in the ra25 directory it moved it to the go bin directory.  And it doesn’t run in either directory, even if the relevant file is in the directory.  But I will throw RA25 away and try again.


On Jan 14, 2020, at 11:51 AM, rcoreilly <notifications@github.com<mailto:notifications@github.com>> wrote:


(base) StephenReadsMBP:~ read$ /Users/read/go/bin/ra25 ; exit;


that looks like you're running the installed executable from the go bin directory -- try running the ./ra25 executable from "go build" in the examples/ra25 dir.
